freqtrade/resolvers/strategy_resolver.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, config: Optional[Dict] = None) -> None:
         # Set attributes
         # Check if we need to override configuration
         # (Attribute name,                              default, experimental)
-        attributes = [("minimal_roi",                   None,    False),
+        attributes = [("minimal_roi",                   {"0": 10.0},    False),
```

Comment:
this is a contrary to your comment on #1645 , although I would vote for not manually formatting the matrix (thus the PR is ok for me) but we should be consistent

Reply:
as you prefer tbh - i can 
* remove all the formatting whitespace
* add whitespace to make this a table again

both work for me ... (maybe I'll just reverse the sequence to have attribute name as last, since that's what's causing the issues ...)